src/main/java/com/puppycrawl/tools/checkstyle/api/FileSetCheck.java

```diff
@@ -30,8 +30,8 @@
     extends Configurable, Contextualizable {
 
     /**
-     * Sets the MessageDispatcher that is used to dispatch error
-     * messages to AuditListeners during processing.
+     * Sets the MessageDispatcher that is used to dispatch audit event
```

Comment:
Here should be plural too.

Reply:
done.

Reply:
@romani this is still singular and not outdated.

Reply:
it was skipped somehow ... strange.
fixed.